packages/patternfly-4/react-core/src/components/Alert/AlertBody.js

```diff
@@ -2,23 +2,32 @@ import React from 'react';
 import { css } from '@patternfly/react-styles';
 import PropTypes from 'prop-types';
 import styles from '@patternfly/patternfly-next/components/Alert/alert.css';
+import { TimesIcon } from '@patternfly/react-icons';
+import { Button, ButtonVariant } from '../Button';
 
 const propTypes = {
   title: PropTypes.node,
   children: PropTypes.node,
-  className: PropTypes.string
+  className: PropTypes.string,
+  onClose: PropTypes.func
 };
 
 const defaultProps = {
   title: null,
   children: '',
-  className: ''
+  className: '',
+  onClose: undefined
 };
 
-const AlertBody = ({ title, className, children, ...props }) => (
+const AlertBody = ({ title, className, children, onClose, ...props }) => (
   <div {...props} className={css(styles.alertBody, className)}>
+    {onClose && (
+      <Button variant={ButtonVariant.plain} onClick={onClose} aria-label="Close">
```

Comment:
Sure thing.

Reply:
@tlabaj this would need to be a prop on `Alert` called something like `closeButtonAriaLabel`, and drilled down to `AlertBody`, because the consumer won't render `AlertBody` directly. Does that sound good or would there be a better name for that prop?